spec/controller_spec.rb

```diff
@@ -5,12 +5,24 @@
 require 'player'
 
 RSpec.describe Controller do
-  describe 'Controller: ' do
+  describe 'Controller: alternates current player after each move ' do
     board = Board.new([1, 2, 3, 4, 5, 6, 7, 8, 9])
     display = Display.new
     player1 = Player.new('x')
     player2 = Player.new('o')
     game = Game.new(board, display, player1, player2)
     controller = Controller.new(game)
+
+    it 'player1 makes a move then current player toggles to player2' do
```

Comment:
Could use context blocks here to make the tests read more like:

context: playing a move when x is the current player, it plays x's move and updates current player.

Reply:
Your example is much clearer and descriptive. I'll add this, thanks Matt.